server/api/personalGoal/personalGoal.controller.js

```diff
@@ -24,7 +22,9 @@ const listPersonalGoals = async (req, res) => {
 }
 
 const updatePersonalGoal = async (req, res) => {
-  const status = req.body.status
+  const { body: pg } = req
```

Comment:
Is there any reason you don't use `const pg = req.body` here?

Reply:
no reason - an earlier version pulled the ability out of the req too.